src/main/webapp/js/common.js

```diff
@@ -532,56 +543,61 @@ function scrollToTop(duration) {
 }
 
 /** Selector for status message div tag (to be used in jQuery) */
-var DIV_STATUS_MESSAGE = '#statusMessage';
+var DIV_STATUS_MESSAGE = '#statusMessagesToUser';
 
 /**
- * Sets a status message. Change the background color to red if it's an error
+ * Sets a status message and the message status.
+ * Default message type is info.
  *
- * @param message
- * @param error
+ * @param message the text message to be shown to the user
+ * @param status the type (StatusType.SUCCESS, StatusType.INFO, StatusType.WARNING, StatusType.DANGER) of the message
```

Comment:
just use @param status type
no need to write down all types as you will need to modify this comment if more status types are added in the future.